api/mraa/gpio.h

```diff
@@ -67,6 +67,7 @@ typedef enum {
     MRAA_GPIOD_ACTIVE_LOW = 4,
     MRAA_GPIOD_OPEN_DRAIN = 5,
     MRAA_GPIOD_OPEN_SOURCE = 6,
+    MRAA_GPIO_DISABLE = 7,
```

Comment:
```suggestion
    MRAA_GPIOD_BIAS_DISABLE = 7,
```
你这是个gpiod独有的功能，不是通用的属性